tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -4796,6 +4780,8 @@ declare namespace ts.server {
         };
     };
     interface ServerHost extends System {
+        watchFile(path: string, callback: FileWatcherCallback, pollingInterval?: number): FileWatcher;
```

Comment:
this is going to be a breaking change to `Session` users. previously these were optional on `ServerHost`, now they are required.

Reply:
But wasnt that implicit requirement? since we use watchFile and watchDirectory anyways without checking if host has it?

Reply:
we do :) ok then. we need to check the jetbrain folks that this is not going to break them